spec-0005/index.md

```diff
@@ -25,6 +26,27 @@ intended use-cases, and pseudo-code illustrating its use.
 Discuss how this would be implemented.
 -->
 
+### Build nightly wheels
+
+By building and uploading wheel for your project you make it easier for projects
+that depend on you to test against your latest changes. They can then report problems
+they find.
+
+There are a few steps to implementing this for your project:
+1. Get access to the area that nightly wheels are uploaded to
+2. Setup a CI step that builds wheels for your project
```

Comment:
Would be nice to link to some doc that teaches people how to build wheel for universal (pure Python) and specific arch (if they have Cython, C-ext, Fortran, etc). Perhaps already some guides out there? _Eyes the cookiecutters group..._

Reply:
Do you know a good guide? I don't unfortunately.

Reply:
I would not hold up the PR on this, after all we're really focusing on the core or big libraries here, if they have complex dependencies, they should have figured out their builds already for normal releases.